packages/synthetix-main/contracts/modules/core/PoolModule.sol

```diff
@@ -114,18 +109,18 @@ contract PoolModule is IPoolModule {
 
             for (
                 ;
-                i < (markets.length < pool.poolDistribution.length ? markets.length : pool.poolDistribution.length);
```

Comment:
Renamed Pool's `poolDistribution` (the array of markets and their weight) to `marketConfigurations`